openCVLibrary330/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
       package="org.opencv"
       android:versionCode="3300"
-      android:versionName="3.3.0">
-
-    <uses-sdk android:minSdkVersion="8" android:targetSdkVersion="21" />
-</manifest>
+      android:versionName="3.3.0" />
```

Comment:
This change is a bit unrelated to the topic of this PR but it's OK.
Next time I'd put it, at least, in a separate commit.

Reply:
It's not, as mentioned in the PR description build tools `3.2.1` added a bunch of warnings and errors. One was an error when SDK versions are specified in the manifest. If I remember right it actually only errors on `targetSdkVersion` and `minSdkVersion` was a warning but I figured removing one would be a less clear change

Reply:
I split it out anyone into https://github.com/farkam135/GoIV/pull/1006, if that gets merged first I'll rebase this branch 👍